### ISSUE TITLE
Feed not loading

### ISSUE THREAD

**Issue by GabriFrz** [j-fbriere/squawker#315]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/j-fbriere/squawker/releases).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/j-fbriere/squawker/issues) or [closed](https://github.com/j-fbriere/squawker/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug, that only occurs in Squawker

### Affected version

3.8.2

### Steps to reproduce the bug

1. Open Squawker 2. Feed not loading same thing with groups

### Expected behavior

Feed loading

### Affected Android/Custom ROM version

Android 13 One UI 5.1

**Comment by knw257** [comment]
I'm running into the same issue. Just downloaded Squawker, signed into my account, imported my subscriptions, and tried to load the feed. Nothing loads. If I try to view the "all" group, the same happens. This also happens with a new group, whether it has all subscriptions (2660 in this case) or just a single one.

**Comment by TheHCJ** [comment]
Is this because you haven't added an account?

**Comment by knw257** [comment]
> Is this because you haven't added an account?

No, I have an account added, however, I think I may have figured it out. A couple times I got an error message indicating rate limiting of the API, and I think since I don't have a paid twitter account, my API access is too restricted for the app to work properly. Unfortunate, but oh well...
